util/personal_names.py

```diff
@@ -32,27 +32,54 @@ def is_corporate_name(display_name):
     return False
 
 
-def display_name_to_sort_name(display_name):
+def display_name_to_sort_name(display_name, advanced=False):
```

Comment:
I was thinking that there might be reasons I don't know about for using the older code, and I wanted to keep it a possibility.  Especially in case there's legacy code I haven't found yet that expects a certain specific response?
Will add unit test.